### ISSUE TITLE
Error in readGEORawFile

### ISSUE THREAD

**Issue by Baiwen224** [kasperdanielhansen/minfi#88]
I'm trying to use minfi's readGEORawFile function to read in methylation data from the GEO  series GSE42752, but I keep getting the following error:

```
gmset=readGEORawFile("GSE42752_methylated_unmethylated_signals.txt",Uname="Signal_A",Mname="Signal_B",sep="\t")

Error in FUN(X[[i]], ...) : 
  assay colnames() must be NULL or equal colData rownames()
```
I tried the example code in refrence manual and get the same error
```

gmset=readGEORawFile("GSE29290_Matrix_Signal.txt",Uname="Signal_A",Mname="Signal_B",sep="\t")

Error in FUN(X[[i]], ...) : 
  assay colnames() must be NULL or equal colData rownames()
```

I'm not very familiar with R and hope that someone could help me with this problem, and thanks for developing the package.

**Comment by kasperdanielhansen** [comment]
you're using minfi devel right?  If so, this is happening due to some
recent changes in minfi.  I thought I had fixed readGEORawFile, but I'll
take a look again.

**Comment by Baiwen224** [comment]
I think I'm using the released version of minfi. I'm not quite sure.
This is my output of sessioninfo(). And thanks again for helping.

```
R version 3.3.2 (2016-10-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.936  LC_CTYPE=Chinese (Simplified)_China.936   
[3] LC_MONETARY=Chinese (Simplified)_China.936 LC_NUMERIC=C                              
[5] LC_TIME=Chinese (Simplified)_China.936    

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] minfi_1.20.0               bumphunter_1.14.0          locfit_1.5-9.1             iterators_1.0.8           
 [5] foreach_1.4.3              Biostrings_2.42.1          XVector_0.14.0 
```            SummarizedExperiment_1.4.0
 [9] GenomicRanges_1.26.1       GenomeInfoDb_1.10.1        IRanges_2.8.1              S4Vectors_0.12.1          
[13] Biobase_2.34.0             BiocGenerics_0.20.0       

loaded via a namespace (and not attached):
 [1] genefilter_1.56.0        splines_3.3.2            lattice_0.20-34          beanplot_1.2            
 [5] rtracklayer_1.34.1       GenomicFeatures_1.26.0   XML_3.98-1.5             survival_2.40-1         
 [9] DBI_0.5-1                BiocParallel_1.8.1       RColorBrewer_1.1-2       registry_0.3            
[13] rngtools_1.2.4           doRNG_1.6                matrixStats_0.51.0       plyr_1.8.4              
[17] pkgmaker_0.22            stringr_1.1.0            zlibbioc_1.20.0          codetools_0.2-15        
[21] memoise_1.0.0            biomaRt_2.30.0           AnnotationDbi_1.36.0     illuminaio_0.16.0       
[25] preprocessCore_1.36.0    Rcpp_0.12.8              xtable_1.8-2             openssl_0.9.5           
[29] limma_3.30.6             base64_2.0               annotate_1.52.0          Rsamtools_1.26.1        
[33] digest_0.6.10            stringi_1.1.2            nor1mix_1.2-2            grid_3.3.2              
[37] GEOquery_2.40.0          quadprog_1.5-5           tools_3.3.2              bitops_1.0-6            
[41] magrittr_1.5             siggenes_1.48.0          RCurl_1.95-4.8           RSQLite_1.1             
[45] MASS_7.3-45              Matrix_1.2-7.1           data.table_1.10.0        httr_1.2.1              
[49] reshape_0.8.6            R6_2.2.0                 mclust_5.2               nlme_3.1-128            
[53] GenomicAlignments_1.10.0 multtest_2.30.0

**Comment by akpelinordor** [comment]
Hi - I am getting the same error when I try readGEORawFile()

I am using this:
`gmset <- readGEORawFile("GSE52576_complete_data.txt", sep="\t", Uname = ".Signal_A", Mname = ".Signal_B", row.names = 1)`

And getting that:
`Read 485577 rows and 57 (of 141) columns from 0.533 GB file in 00:00:05`
`Error in FUN(X[[i]], ...) : assay colnames() must be NULL or equal colData rownames()`

Any feedback? Best, A/

**Comment by schiacchia** [comment]
Dear akpelinordor and others,

Did you ever manage to resolve this issue? If so, would you mind posting a quick synopsis of the solution? 

Best,
S

**Comment by kasperdanielhansen** [comment]
This has - I hope - been addressed by minfi 1.25.1 which I just updated on Github and submitted to Bioconductor. Since this is a bugfix I am planning to release minfi 1.24.1 on the stable branch of the latest Bioconductor release.
